packages/libgsf.rb

```diff
@@ -1,6 +1,6 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Libgsf < Package
+class Libgsf < Autotools
   description 'The G Structured File Library'
   homepage 'https://gitlab.gnome.org/GNOME/libgsf'
   version '1.14.50'
```

Comment:
Update to [1.14.52](https://gitlab.gnome.org/GNOME/libgsf/-/tags/LIBGSF_1_14_52)?